### PR TITLE
feat: ImageCard extension

### PR DESCRIPTION
**Pull request by Lakate**
https://github.com/gravity-ui/page-constructor/issues/875

### CODE REVIEW

File: src/sub-blocks/ImageCard/ImageCard.tsx [hunk 1]
```diff
@@ -20,16 +22,23 @@ const ImageCard = (props: ImageCardProps) => {
         direction = ImageCardDirection.Direct,
         margins,
         backgroundColor,
+        url,
+        target,
+        urlTitle,
+        additionalInfo,
+        links,
+        buttons,
+        list,
```
Comment (gorgeousvlad) [inline src/sub-blocks/ImageCard/ImageCard.tsx:31]: `...contentProps`

File: src/sub-blocks/ImageCard/ImageCard.tsx [hunk 2]
```diff
@@ -38,9 +47,43 @@ const ImageCard = (props: ImageCardProps) => {
...
-                    <Content title={title} text={text} colSizes={{all: 12, md: 12}} size="s" />
+                    <Content
+                        titleId={titleId}
+                        title={title}
+                        text={text}
+                        links={links}
+                        buttons={buttons}
+                        list={list}
+                        theme={cardTheme}
+                        additionalInfo={additionalInfo}
```
Comment (gorgeousvlad) [inline src/sub-blocks/ImageCard/ImageCard.tsx:58]: `{...contentProps}`

File: src/models/constructor-items/sub-blocks.ts [hunk 3]
```diff
@@ -188,12 +188,18 @@ export interface LayoutItemProps extends ClassNameProps, CardLayoutProps, Analyt
...
+export interface ImageCardProps
+    extends CardBaseProps,
+        CardLayoutProps,
+        Omit<ContentBlockProps, 'colSizes' | 'centered'> {
     image: ImageProps;
     enableImageBorderRadius?: boolean;
     margins?: ImageCardMargins;
     direction?: ImageCardDirection;
     backgroundColor?: string;
+    url?: string;
```
Comment (gorgeousvlad) [inline src/models/constructor-items/sub-blocks.ts:200]:
```
link: Pick<LinkProps, 'url' | `urlTitle` | `target`> 
```

**Review by gorgeousvlad — CHANGES_REQUESTED** [review]

File: src/models/constructor-items/sub-blocks.ts (hunk 3 above)
Comment (Lakate) [reply src/models/constructor-items/sub-blocks.ts:200]: I use these props to have similar interface with other cards, for example 
- https://github.com/gravity-ui/page-constructor/blob/main/src/sub-blocks/BasicCard/schema.ts
- https://github.com/gravity-ui/page-constructor/blob/main/src/sub-blocks/BackgroundCard/schema.ts

**Pull request merged**